drivers/interrupt_controller/Kconfig.multilevel

```diff
@@ -0,0 +1,118 @@
+# Kconfig - Multilevel interrupt configuration
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config MULTI_LEVEL_INTERRUPTS
+	bool "Multi-level Interrupts"
```

Comment:
Should we express that this depends on GEN_ISR_TABLES?

We don't want to be exposing this to platforms that don't use gen_isr_tables.

Reply:
agreed.